app/src/main/java/com/eventyay/organizer/core/event/create/UpdateEventFragment.java

```diff
@@ -109,7 +109,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         setHasOptionsMenu(true);
 
-        binding.form.eventOriginalImageLayout.setOnClickListener(view -> {
+        binding.form.eventOriginalImage.setOnClickListener(view -> {
```

Comment:
Unnecessary change

Reply:
Sorry sir , this was the change from the another pull request I have made whose commits got mixed with it .I will fix it